### DISCUSSION TITLE
SCI depends on hardware and usage

### DISCUSSION THREAD

**Discussion by ElmarBorgmeier** [Green-Software-Foundation/sci#244] [SCI feedback]
Academic papers on Green Software commonly stress the rather obvious fact that software does not create emissions directly. Instead, it induces them via the required hadware and CPU cycles.

This shows up in the SCI definition clearly enough. We should, however, in my opinion nonetheless express our standpoint concerning the implications.

When measuring an SCI, the results depend on:

- the environment (Server, PUE, ...)
- the usage scenario (How is the software used? In practise: Which scripts have been used to simulate software behaviour?)   

Any changes to these parameters change the SCI. So, to be precise, the SCI does not measure a property of the software as such. It measures a software system (=SW+HW) that ist used in an assumed way.

How do we deal with this?

**Comment by ElmarBorgmeier** [comment]
My point of view: We should encourage the publication of two versions of an SCI:

- A "Software System SCI" based on figures from the actual measurement environment and usage scenario. Environment and scenario should be published as an addendum ("small print") along with the SCI.
- A "Default SCI" based on the same measurements, but calculated using default values for both the conversion from energy to emissions and for embodied emission of specific types of hardware (CPU, Memory, disk space, internet use,...).

Why? Because, sometimes one version will be more useful and sometimes the other. Examples:

A) I am considering using SaaS. In this case, the environment is known. The "Actual SCI" is best for me.
B) I consider purchasing a Software, which will be installed in an environment that differs from the one the SCI measurements have been made on. In this case, the "default SCI" is more suitable.

**Reply by atg-abhishek** [reply]
Interesting formulation, let's chat more about it on the WG call.

**Comment by Henry-WattTime** [comment]
WG: Henry fixing his pull request/branch that addresses this.

Elmar: Normalized/default SCI? -Future 
Gadhu: Buying service, don't have much leverage. Good/bad score, getting a score is the only choice that can be made. But if buying software and running on own hardware, then can make more granular decisions 

Discuss later after case studies?

**Reply by ElmarBorgmeier** [reply]
I agree to discussing it later.
